lib/core/src/lib/form/components/widgets/date-time/date-time.widget.spec.ts

```diff
@@ -101,7 +107,7 @@ describe('DateTimeWidgetComponent', () => {
         });
 
         widget.field = field;
-        const mockDate = moment('1982-03-13T10:00:000Z', 'YYYY-MM-DDTHH:mm:ssZ');
+        const mockDate = DateFnsUtils.formatDate(new Date('1982-03-13 10:00 AM'), 'YYYY-MM-DDTHH:mm:ssZ');
```

Comment:
Incorrect format for `new Date`, use the original `'1982-03-13T10:00:000Z'`

Reply:
using '1982-03-13T10:00:000Z' now